### ISSUE TITLE
[Bug]: app goes to background immediately

### ISSUE THREAD

**Issue by elpaso** [MarmadileManteater/FreeTubeCordova#327]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Install the app from F-Droid after adding the repo
2. run the app
3. search for "Sting"
4. click on first result
5. play first video result
6. video starts buffering
7. the app screen disappears, no video or sound is played
8. two notifications: "Freetube is running in the background" and another one with the player control symbols "<| || >|"
9. clicking on the notifications has no effect

### Expected Behavior

the video plays

### Issue Labels

content not loading

### FreeTube Version

v0.20.0.11 Beta

### Operating System Version

android 10

### Installation Method

.apk

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by MarmadileManteater** [comment]
It sounds like the app is crashing.

**Comment by MarmadileManteater** [comment]
The notification doesn't get cancelled because the lifecycle hook isn't called when the app crashes like that, but the app is not still running. Well, the service probably is, but the activity is not.

**Comment by elpaso** [comment]
Yes, the app is crashing, after a while the OS tells me that the app is repeatedly crashing. Is there anything I can do to debug this?

**Comment by MarmadileManteater** [comment]
You have to download [the nightly](https://github.com/MarmadileManteater/FreeTubeCordova/actions/runs/8583493074) (release is no longer debuggable), and you should be able to read the logs using `logcat` _(although, you might be able to read logcat for release, idk, but if you actually want to debug and step through the code, you need nightly)_. I would personally use android studio (since I already have it), but it might be easier to just get the [SDK platform tools](https://developer.android.com/tools/releases/platform-tools) and run `adb logcat`.

Typically, most problems can be debugged with the chrome inspector, but this problem doesn't seem like it would be easy to debug this way seeing as the problem would definitely result in the webview being destroyed, closing the debug session.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@elpaso cant reproduce this on v0.23. Please update to the latest version and let us know if this is still happening on your end
